src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const partner = person.sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {person.name}</h2>
+
+      {person.age && <p className="Person__age">I am {person.age}</p>}
+
+      {person.isMarried ? (
+        <p className="Person__partner">
```

Comment:
In both cases `<p className="Person__partner">` the same, only a content is different